### PR TITLE
Data Science with Python: Introduction to Keras

### PR DESCRIPTION
**Pull request by freny24**
<hr>

## Description 📜

Explained basics of Keras using code snippets [#416 ]

<hr>

## Type of change

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Datascience with Python

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

### CODE REVIEW

**Comment by prathimacode-hub** [comment]
Your README and .ipynb file both lack details. Be elaborative. And follow the project structure mentioned. Make sure to note files needed for a successful PR. This is not sufficient. @freny24

File: Datascience_With_Python/Deep Learning/DSWP/Intoduction_to_Keras.ipynb [hunk 1]
```diff
@@ -0,0 +1,1246 @@
...
+      "Requirement already satisfied: requests-oauthlib>=0.7.0 in e:\\anaconda3\\lib\\site-packages (from google-auth-oauthlib<0.5,>=0.4.1->tensorboard~=2.6->tensorflow) (1.3.0)\n",
+      "Requirement already satisfied: importlib-metadata; python_version < \"3.8\" in e:\\anaconda3\\lib\\site-packages (from markdown>=2.6.8->tensorboard~=2.6->tensorflow) (1.5.0)\n",
+      "Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in e:\\anaconda3\\lib\\site-packages (from pyasn1-modules>=0.2.1->google-auth<2,>=1.6.3->tensorboard~=2.6->tensorflow) (0.4.8)\n",
+      "Requirement already satisfied: oauthlib>=3.0.0 in e:\\anaconda3\\lib\\site-packages (from requests-oauthlib>=0.7.0->google-auth-oauthlib<0.5,>=0.4.1->tensorboard~=2.6->tensorflow) (3.1.1)\n",
+      "Requirement already satisfied: zipp>=0.5 in e:\\anaconda3\\lib\\site-packages (from importlib-metadata; python_version < \"3.8\"->markdown>=2.6.8->tensorboard~=2.6->tensorflow) (2.2.0)\n"
+     ]
+    }
+   ],
+   "source": [
+    "! pip install tensorflow"
```
Comment (Kareem-negm) [inline Datascience_With_Python/Deep Learning/DSWP/Intoduction_to_Keras.ipynb:139]: if you hide this cell output it's will be nice

File: Datascience_With_Python/Deep Learning/DSWP/Intoduction_to_Keras.ipynb [hunk 2]
```diff
@@ -0,0 +1,1246 @@
...
+      "4236    86.0\n",
+      "4237    80.0\n",
+      "4238    67.0\n",
+      "4239    85.0\n",
+      "Name: heartRate, Length: 4240, dtype: float64\n"
+     ]
+    }
+   ],
+   "source": [
+    "print(x)\n",
```
Comment (Kareem-negm) [inline Datascience_With_Python/Deep Learning/DSWP/Intoduction_to_Keras.ipynb:1102]: try to add this "print " as a DF output to be more readable

**Review by Kareem-negm — COMMENTED** [review]
your work is good for now just keep doing and good luck

File: Datascience_With_Python/DS Datasets/README.md [hunk 3]
```diff
@@ -1 +1,4 @@
-
+## Dataset used:
+     I used Heart rate prediction CSV file to predict heart rate among various patients using Tensorflow and Keras. I downloaded the data set from Kaggle
```
Comment (Kareem-negm) [inline Datascience_With_Python/DS Datasets/README.md:2]: it's shown as a scroll bar, please check it again

**Review by prathimacode-hub — CHANGES_REQUESTED** [review]
Changes to be done 👍 

The project files of .ipynb and main README.md file should be in "/Datascience_with_Python/Deep Learning/Tutorials/Introduction to Keras/" repo
The file name of .ipynb file should be in snake case and README.md file
And datasets should be added in "/Datascience_with_Python/DS Datasets/Introduction to Keras/" repo

Kindly work on the project structure changes. @freny24

**Pull request closed without merging**